src/plugins/ConditionalRule.js

```diff
@@ -18,6 +19,7 @@ export default class ConditionalRule {
 
   constructor(selector: string, styles: Object, options: RuleOptions) {
     this.selector = selector
+    this.selectorId = selector.replace(' ', '')
```

Comment:
but we need id without spaces for keyframes too, because CSSKeyframesRule does not have `selectorText` too